drivers/input/Kconfig

```diff
@@ -30,17 +30,23 @@ config MOUSE_WHEEL
 
 endif # MOUSE
 
+config TOUCHSCREEN
```

Comment:
should we apply the similar change to MOUSE and JOYSTICK?

Reply:
Yes, it makes sense to extend this approach to the other input classes.
For JOYSTICK should have no impact, since CONFIG_JOYSTICK currently does not exist.
But CONFIG_MOUSE will have an impact to existing configurations, because it will change the current behavior.
So my recommendation is that we address each one in a dedicated PR.

Reply:
Ok, it's reasonable. @btashton could you take a look?